test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2042,6 +2038,13 @@ class test_ucp_sockaddr_protocols : public test_ucp_sockaddr {
 
         do {
             short_progress_loop();
+
+            ucs_status_t send_status = request_check_status(sreq);
+            if ((send_status != UCS_INPROGRESS) &&
+                !check_send_status(send_status, receiver(), NULL, cb_type())) {
+                return NULL;
```

Comment:
why need to use check_send_status() here? we don't need to cancel the receive request.
maybe just return NULL if send_status is UNREACHABLE?